_docker/environments/drupal-pull-request/rhd.settings.yml.erb

```diff
@@ -10,9 +10,9 @@ drupal:
   port: "<%=ENV['DRUPAL_HOST_PORT']%>"
 searchisko:
   protocol: 'http'
```

Comment:
@paulrobinson Can we make this HTTPS and port 443 so that the PR environment supports HTTPS?